sandy/core/endf6.py

```diff
@@ -2088,3 +2089,167 @@ def get_gendf(self,
             if to_file:
                 shutil.move(gendf_file, to_file)
         return groupr
+
+    def get_lpc_cov(self, **kwargs):
+        """
+        Extract global Legendre Polynomials coefficients covariance matrix
+        from `sandy.Endf6` instance.
+
+        Parameters
+        ----------
+        mat : `int`, optional
+            MAT number. The default is None.
+        mt : `int`, optional
+            MT number. The default is None.
```

Comment:
it is weird to have parameters here and not in the initialization line.
Couldn't you do `get_lpc_cov(self, mat=range(1, 10000), mat=range(1, 1000))`